render/egl.c

```diff
@@ -185,6 +181,22 @@ struct wlr_egl *wlr_egl_create_with_drm_fd(int drm_fd) {
 	load_egl_proc(&egl->procs.eglGetPlatformDisplayEXT,
 		"eglGetPlatformDisplayEXT");
 
+	if (check_egl_ext(client_exts_str, "EGL_KHR_platform_gbm")) {
```

Comment:
Can be simplified to `egl->exts.KHR_platform_gbm = check_egl_ext(…)`

Reply:
Added check for `EGL_EXT_platform_device` as well